src/Sulu/Component/Webspace/Tests/Unit/Manager/PhpWebspaceCollectionDumperTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sulu\Component\Webspace\Tests\Unit\Manager;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Sulu\Component\Webspace\Manager\Dumper\PhpWebspaceCollectionDumper;
+use Sulu\Component\Webspace\Manager\WebspaceCollection;
+
+class PhpWebspaceCollectionDumperTest extends TestCase
+{
+    use ProphecyTrait;
+
+    /** @var WebspaceCollection|MockObject */
+    private $webspaceCollection;
+
+    private PhpWebspaceCollectionDumper $webspaceDumper;
```

Comment:
looks like php 7.2 does not like this:

```suggestion
     /**
      * @var PhpWebspaceCollectionDumper
      */
    private $webspaceDumper;
```

Reply:
Good old php7.2 :D